src/bsp.rs

```diff
@@ -1,31 +1,18 @@
-use crate::{is_zero, Intersection, Plane, Polygon, Splitter};
+use crate::{is_zero, Intersection, Plane, Polygon};
 
 use euclid::default::{Point3D, Vector3D};
 
 use std::{fmt, iter};
 
-/// A plane abstracted to the matter of partitioning.
-pub trait BspPlane: Sized + Clone {
-    /// Try to cut a different plane by this one.
-    fn cut(&self, other: Self) -> PlaneCut<Self>;
-    /// Check if a different plane is aligned in the same direction
-    /// as this one.
-    fn is_aligned(&self, otgher: &Self) -> bool;
-}
-
-impl<A> BspPlane for Polygon<A>
+impl<A> Polygon<A>
 where
-    A: Copy + fmt::Debug,
+    A: Copy,
 {
-    fn cut(&self, mut poly: Self) -> PlaneCut<Self> {
-        log::debug!("\tCutting anchor {:?} by {:?}", poly.anchor, self.anchor);
-        log::trace!("\t\tbase {:?}", self.plane);
```

Comment:
nitpick: I don't see an issue, but _technically_ the commit removing the `log` statements doesn't note that it's doing so.